_blog/2023-12-15-using-actix-rust.mdx

```diff
@@ -350,26 +388,29 @@ where
     }
 }
 ```
+
 Now that we've written our middleware, we can now add it to our App:
+
 ```rust
 #[actix_web::main]
 async fn main() -> std::io::Result<()> {
-// access logs are printed with the INFO level so ensure it is enabled by default
-env_logger::init_from_env(env_logger::Env::new().default_filter_or("info"));
+    let app = App::new()
+         .wrap(SayHi);
```

Comment:
Should this be `SayHiMiddleware`?

Reply:
No, but good eye; this snippet is wrong.

There's not middleware builder type called `SayHi` yet. (See code block where this was copied from: https://actix.rs/docs/middleware)

Reply:
The `SayHi` struct is declared further up in the article, should we maybe change this so it's clear that both are related?

Reply:
ahh so it is, i think its acceptable tbh, someone following along with the flow of the artcle will have it in place